blog.html

```diff
@@ -26,6 +26,16 @@ <h2>Project News and Blog</h2>
 
 {% for post in site.posts %}
   {% unless post.categories contains 'translation' %}
-    {% include blog_entry.html %}
+  <p>
+    <h3>
+      <a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a>
+    </h3>
+    <p>
+    <span class="blog-list-date">
+      {{ post.date | date: "%-d %B %Y" }}
+    </span>
```

Comment:
What do you think about adding the author here also?

Reply:
I tried it and preferred how it looked without. Most say "The Apache Arrow PMC (pmc)" so it seemed like noise.